README.md

```diff
@@ -651,6 +651,30 @@ helper.getCurrentPage();
 helper.setCurrentPage(3).search();
 ```
 
+#### Automatic reset to page 0
+
+During a search, changing the parameters will update the result set, which can then change
+the number of pages in the result set. Therefore, the behavior has been standardized so
+that any operation that may change the number of page will reset the pagination to page 0.
+
+This may lead to some unexpected behavior. For example:
+
+```js
+// helper.state.page = 0
+helper.setPage(4);
+// helper.state.page = 4
+helper.setQuery('foo');
+// helper.state.page = 0
+```
+
+Non exhaustive list of operations that trigger a reset :
```

Comment:
Non exhaustive because there are two many of them? Maybe we can do the opposite list (or it's not needed?)